src/project_settings.py

```diff
@@ -22,6 +22,23 @@
             # ignore - empty person
             198
         ],
+        # clarin-dspace=# select * from metadatafieldregistry  where metadata_field_id=176 ;
+        #  metadata_field_id | metadata_schema_id |   element   |   qualifier   |               scope_note
+        # -------------------+--------------------+-------------+---------------+----------------------------------------
+        #                176 |                  3 |  bitstream  |     file      | Files inside a bitstream if an archive
+        # clarin-dspace=# select * from metadatafieldregistry  where metadata_field_id=178 ;
+        # -------------------+--------------------+-------------+---------------+----------------------------------------
+        #                178 |                  3 |  bitstream  | redirectToURL |    Get the bitstream from this URL.
+        "fields": [176, 178]
```

Comment:
this does not seem to be independent of metadata field ids, use keys instead?!

Reply:
Those are fields we want to ignore, not replace